contract-tools/xchain/xchain.go

```diff
@@ -483,36 +494,39 @@ const (
 	// libp2p identifier for latest deal protocol
 	DealProtocolv120 = "/fil/storage/mk/1.2.0"
 	// Delay to start deal at. For 2k devnet 4 second block time this is 13.3 minutes TODO Config
-	dealDelayEpochs = 200
+	dealDelayEpochs = 10000
 	// Storage deal duration, TODO figure out what to do about this, either comes from offer or config
 	dealDuration = 518400 // 6 months (on mainnet)
 )
 
 func (a *aggregator) runAggregate(ctx context.Context) error {
 	// pieces being aggregated, flushed upon commitment
 	// Invariant: the pieces in the pending queue can always make a valid aggregate w.r.t a.targetDealSize
-	pending := make([]DataReadyEvent, 0, 256)
+	// pending := make([]DataReadyEvent, 0, 256)
+	var pending []DataReadyEvent
 	total := uint64(0)
-	prefixPiece := filabi.PieceInfo{
-		Size:     filabi.PaddedPieceSize(prefixCARSizePadded),
-		PieceCID: cid.MustParse(prefixCARCid),
-	}
+	// prefixPiece := filabi.PieceInfo{
```

Comment:
Since this appears not to be necessary I think you should delete this

cc @ribasushi apparently boost doesn't hate us as much as we thought?